content/sensu-go/5.0/reference/assets.md

```diff
@@ -11,25 +11,43 @@ menu:
     parent: reference
 ---
 
-- [Specification](#assets-specification)
+- [What is an asset?](#what-is-an-asset)
+- [How do assets work?](#how-do-assets-work)
+- [Asset format specification](#asset-format-specification)
+- [Sharing an asset on Bonsai](#sharing-an-asset-on-bonsai)
+- [Asset specification](#asset-specification)
 - [Examples](#examples)
 
+You can discover, download, and share assets using [Bonsai, the Sensu asset index][16].
+Read the [guide to using assets][23] to get started.
+
 ## What is an asset?
-An asset is an executable that a check, handler, or mutator can specify as a
-dependency. Assets must be a tar archive (optionally gzipped) with scripts or
-executables within a bin folder. At runtime, the backend or agent installs
-required assets using the specified URL. Assets let you manage runtime
-dependencies without using configuration management tools.
+Assets are shareable, reusable packages that make it easy to deploy Sensu plugins.
+You can use assets to provide the plugins, libraries, and runtimes you need to automate your monitoring workflows.
+Sensu supports runtime assets for [checks][6], [filters][7], [mutators][8], and [handlers][9].
 
 ## How do assets work?
-Assets can be executed by the backend (for handler and mutator assets), or
+Assets can be executed by the backend (for handler, filter, and mutator assets), or
 by the agent (for check assets). At runtime, the entity sequentially fetches
 assets and stores them in its local cache. Asset dependencies are then
 injected into the `PATH` so they are available when the command is executed.
-Subsequent check, handler, or mutator executions look for the asset in local
+Subsequent check, handler, filter, or mutator executions look for the asset in local
 cache and ensure the contents match the checksum. An entity's local cache can
```

Comment:
While not part of this change, this may need an update. It talks about `An entity's local cache`. Do we consider a sensu backend an entity? It can be one _if_ you have an agent. I bring this up as a sensu backend can have assets and if they aren't considered an entity, we may want to say something like `An agent or backend's local cache` instead.

I don't see the `cache-dir` flag in the backend reference. So are all assets downloaded by an entity when they are required? If so, that would mean assets for use on the backend are dependent on an agent to do that work and that should definitely be called out somewhere if it isn't already.

Reply:
Great catch! I've updated the wording here and opened a PR to add `cache-dir` to the backend reference.